spec/models/topic_embed_spec.rb

```diff
@@ -41,8 +41,12 @@
       end
 
       it "Supports updating the post" do
-        post = TopicEmbed.import(user, url, title, "muhahaha new contents!")
+        new_user = Fabricate(:user)
+
+        post = TopicEmbed.import(new_user, url, title, "muhahaha new contents!")
```

Comment:
don't we want to test the title is updated as well? will merge this in and add that.